plugins/data_migration/api/data_migration_helper.js

```diff
@@ -983,7 +987,48 @@ module.exports = function(my_db) {
                     run_command(command.cmd, command.args);
                 }).then(
                     function() {
-                        resolve();
+                        //update messages 
+                        if (mydata && mydata.app_ids) {
+                            log_me(logpath, 'Updating records in messages collection', false);
+                            var imported_ids = mydata.app_ids.split(',');
+                            var objectIDS = [];
+                            for (let z = 0; z < imported_ids.length; z++) {
+                                if (imported_ids[z] !== "") {
+                                    objectIDS.push(common.db.ObjectID(imported_ids[z]));
+                                }
+                            }
+                            common.db.collection('messages').find({"apps": {"$in": objectIDS}}).toArray(function(err1, list) {
+                                if (err1) {
+                                    log_me(logpath, err1, false);
+                                }
+                                for (let z = 0; z < list.length; z++) {
+                                    if (list[z].auto === true) {
+                                        if ((list[z].result.status & 2) > 0) {
+                                            list[z].result.status = list[z].result.status & ~2;
+                                            common.db.collection('messages').update({_id: list[z]._id}, { $set: {'result.status': list[z].result.status}}, function(err/*, res*/) {
+                                                if (err) {
+                                                    log_me(logpath, err, false);
+                                                }
+                                            });
+                                        }
+                                    }
+                                    else if (list[z].auto !== true) {
+                                        if ((list[z].result.status & 2) > 0) {
+                                            list[z].result.status = list[z].result.status & ~2 | 16 | 1024;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of '|'.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=5091567)